styles/styles.css

```diff
@@ -0,0 +1,1029 @@
+* {
+    box-sizing: border-box;
+    margin: 0;
+    padding: 0;
+}
+
+body {
+    font-family: "Roboto Condensed", Arial, Helvetica, sans-serif;
+}
+
+/* NAVBAR */
+
+.first-nav {
+    display: none;
+}
+
+.top-nav {
+    align-items: flex-end;
+    background-color: #f72210;
+    display: flex;
+    height: 90px;
+    justify-content: space-between;
+    width: 100%;
+}
+
+.top-nav p {
+    color: white;
+    font-family: "Roboto Condensed";
+    font-size: 16px;
+    margin-left: 20px;
+}
+
+.top-nav img {
+    height: 60px; 
+    margin-bottom: 10px;
+    width: 396px;
+}
+
+.btns {
+    margin-bottom: 20px;
+    margin-right: 30px;
+}
+
+.btn-sign {
+    background-color: #f72210;
+    color: white;
+    font-family: "Roboto Condensed";
+    font-size: 18px;
+    font-weight: bold;
+    height: 26px;
+    line-height: 14px;
+    padding: 6px 12px;
+    width: 80px;
+}
+
+.btn-subs {
+    background-color: white;
+    color: #f72210;
+    font-family: "Roboto Condensed";
+    font-size: 18px;
+    font-weight: bold;
+    height: 26px;
+    line-height: 14px;
+    padding: 6px 10px 6px 12px;
+    width: 128px;
+}
+
+.bottom-nav {
+    border-bottom: 1px solid lightgray;
+    height: 56px;
+    padding: 10px 0;
+    width: 100%;
+}
+
+.bottom-nav li {
+    display: inline;
+    padding: 22px;
+}
+
+ .bottom-nav li::after {
+    color: lightgray;
+    content: "|";
+    font-size: 25px;
+    left: 25px;
+    position: relative;
+  } 
+
+.bottom-nav a {
+    color: black;
+    font-family: "Roboto Condensed";
+    font-size: 21px;
+    font-weight: bold;
+    width: 93px;
+}
+
+.bottom-nav a:hover {
+    color: red;
+    text-decoration: none;
+}
+.src {
+    color: #333333;
+    font-family: Arial;
+    font-size: 16px;
+}
+
+.fa-search {
+    margin-left: 10px;
+}
+
+/* SECOND NAVBAR */
+
+.second-navbar {
+    align-items: center;
+    background-color: red;
+    display: flex;
+    height: 50px;
+    justify-content: space-between;
+    position: fixed;
+    width: 100%;
+    z-index: 2;
+}
+
+.second-navbar img {
+    height: 20px;
+    margin-left: 20px;
+    width: 132px;
+}
+
+.btns-2 { 
+    margin-right: 30px;
+}
+
+.btn-1, .btn-2 {
+    position: relative;
+    bottom: 8px;
+}
+
+.fa-bars {
+    color: #333333;
+    cursor: pointer;
+    font-size: 36px;
+    margin-left: 10px;
+    opacity: 0.5;
+}
+
+/* MAIN SECTION */
+
+ /*.main-container {
+   
+} */
+
+.main-columns {
+    display: flex;
+    justify-content: center;
+}
+
+/* JUST TO SEE THE COLUMNS */
+.first-column {
+    /*background-color: aquamarine;*/
```

Comment:
Consider deleting commented out codes. If you need to review a previous version of your code, use git history.